src/terminal/terminaldisplayinit.cc

```diff
@@ -36,72 +36,72 @@
 #include "config.h"
 #include "terminaldisplay.h"
 
-#include <string>
 #include <stdexcept>
+#include <string>
 
 #if defined HAVE_NCURSESW_CURSES_H
-#  include <ncursesw/curses.h>
-#  include <ncursesw/term.h>
+#include <ncursesw/curses.h>
+#include <ncursesw/term.h>
 #elif defined HAVE_NCURSESW_H
-#  include <ncursesw.h>
-#  include <term.h>
+#include <ncursesw.h>
+#include <term.h>
 #elif defined HAVE_NCURSES_CURSES_H
-#  include <ncurses/curses.h>
-#  include <ncurses/term.h>
+#include <ncurses/curses.h>
+#include <ncurses/term.h>
 #elif defined HAVE_NCURSES_H
-#  include <ncurses.h>
-#  include <term.h>
+#include <ncurses.h>
+#include <term.h>
 #elif defined HAVE_CURSES_H
-#  include <curses.h>
-#  include <term.h>
+#include <curses.h>
+#include <term.h>
 #else
-#  error "SysV or X/Open-compatible Curses header file required"
+#error "SysV or X/Open-compatible Curses header file required"
 #endif
 #include <stdlib.h>
 #include <string.h>
 
 using namespace Terminal;
 
-static bool ti_flag( const char *capname )
-{
+static bool ti_flag( const char *capname ) {
   int val = tigetflag( const_cast<char *>( capname ) );
   if ( val == -1 ) {
-    throw std::invalid_argument( std::string( "Invalid terminfo boolean capability " ) + capname );
+    throw std::invalid_argument(
+        std::string( "Invalid terminfo boolean capability " ) + capname );
   }
   return val;
 }
 
-static const char *ti_str( const char *capname )
-{
+static const char *ti_str( const char *capname ) {
   const char *val = tigetstr( const_cast<char *>( capname ) );
   if ( val == (const char *)-1 ) {
-    throw std::invalid_argument( std::string( "Invalid terminfo string capability " ) + capname );
+    throw std::invalid_argument(
+        std::string( "Invalid terminfo string capability " ) + capname );
   }
   return val;
 }
 
 Display::Display( bool use_environment )
-  : has_ech( true ), has_bce( true ), has_title( true ), smcup( NULL ), rmcup( NULL )
-{
+  : has_ech( true ),
+    has_bce( true ),
+    has_title( true ),
+    smcup( NULL ),
+    rmcup( NULL ) {
   if ( use_environment ) {
     int errret = -2;
     int ret = setupterm( (char *)0, 1, &errret );
 
     if ( ret != OK ) {
       switch ( errret ) {
       case 1:
-	throw std::runtime_error( "Terminal is hardcopy and cannot be used by curses applications." );
-	break;
-      case 0:
-	throw std::runtime_error( "Unknown terminal type." );
-	break;
+        throw std::runtime_error(
+            "Terminal is hardcopy and cannot be used by curses applications." );
+        break;
+      case 0: throw std::runtime_error( "Unknown terminal type." ); break;
```

Comment:
I think the newlines here should also be persisted

Reply:
Ditto.